docker-compose.yml

```diff
@@ -9,6 +9,6 @@ services:
     ports:
       - "10137:10137"
     environment:
-      PROFILE_URI: https://127.0.0.1:9010/v1
+      PROFILE_URI: https://127.0.0.1:1111/profile/v1
```

Comment:
side nite: the profile uri looks wrong for local config. the content server repo or others have the correct one